$NetBSD: patch-ac,v 1.3 2012/04/16 04:01:10 dholland Exp $

- Use standard headers.
- Don't provide own definitions of now-standard symbols.

--- tune.h.orig	Mon Sep 25 23:22:26 2000
+++ tune.h
@@ -6,11 +6,10 @@
 
 #define BYTESIZ		8
 
-#ifdef SVID
 #include <unistd.h>
-#endif
+#include <string.h>
 
-#ifdef BSD42
+#if 0
 #define SEEK_SET	L_SET
 #define	memset(s,c,n)	bzero(s, n)		/* only when c is zero */
 #define	memcpy(s1,s2,n)	bcopy(s2, s1, n)
